controllers/flowlogspipeline/flp_objects.go

```diff
@@ -255,10 +258,70 @@ func (b *builder) addTransformStages(lastStage *config.PipelineBuilderStage) {
 	agg := enrichedStage.Aggregate("aggregate", []api.AggregateDefinition{})
 	agg.EncodePrometheus("prometheus", api.PromEncode{
 		Port:   int(b.desired.PrometheusPort),
-		Prefix: "flp_",
+		Prefix: "netobserv_",
```

Comment:
Totally agree with this change, but to let @memodi know: once PR is merged, FLP metrics will have a `netobserv_` prefix instead of `flp_`. So you'll probably have a couple of tests/docs/dashboards to update